lib/urlwatch/command.py

```diff
@@ -212,6 +212,31 @@ def modify_urls(self):
             print('Adding %r' % (job,))
             self.urlwatcher.jobs.append(job)
 
+        if self.urlwatch_config.change_location is not None:
+            new_loc = self.urlwatch_config.change_location[1]
+            # Ensure the user isn't overwriting an existing job with the change.
+            if new_loc in (j.get_location() for j in self.urlwatcher.jobs):
+                print('The new location "%s" already exists for a job. Use that '
+                      'existing job or choose a different value.' % new_loc)
```

Comment:
Use [f-strings](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) instead of `%`-formatting.